src/PVRIptvData.cpp

```diff
@@ -469,12 +470,21 @@ bool PVRIptvData::LoadPlayList(void)
         tmpChannel.properties.insert({prop, propValue});
       }
     }
+    else if (StringUtils::Left(strLine, strlen(PLAYLIST_TYPE_MARKER)) == PLAYLIST_TYPE_MARKER)
+    {
+      std::string value = ReadMarkerValue(strLine, PLAYLIST_TYPE_MARKER);
+      if (value == "VOD")
```

Comment:
nitpick: you could eliminate the variable here; `if ReadMarkerValue(strLine, PLAYLIST_TYPE_MARKER) == "VOD"` also would do the trick.

Reply:
Sure it's a leftover cause I had 2 if's before :)